### PR TITLE
badge component completed

### PR DESCRIPTION
**Pull request by Sharmaryan**
Screenshot (25)
Screenshot (26)

badge component done, not responsive

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -10,12 +10,12 @@
   box-sizing: border-box;
 }
 
-nav {
+/* nav {
```
Comment (sonishreyas) [inline style.css]: remove this code if not needed

**Review by sonishreyas — APPROVED** [review]
Looks good to merge

**Review by nyyshaaa — APPROVED** [review]
Looks good to me!

File: styles/badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+.badges-offline {
+  background-color: var(--status-offline);
+}
+.badges-dnd {
+  background-color: var(--status-dnd);
+}
+.icon-size{
+font-size: 3rem;
+}
+.icon-msg{
```
Comment (TejasShekar) [inline styles/badges/badges.css:51]: add some padding so that numbers inside badge looks clean

**Review by TejasShekar — APPROVED** [review]
Suggested one change. 
Rest looks good to merge.

File: styles/badges/badges.css (hunk 2 above)
Comment (Sharmaryan) [reply styles/badges/badges.css:51]: done

File: style.css (hunk 1 above)
Comment (Sharmaryan) [reply style.css]: done

**Pull request merged**